docs/releases/1.10.1.txt

```diff
@@ -61,3 +61,7 @@ Bugfixes
 
 * Fixed ``contrib.admindocs`` crash when a view is in a class, such as some of
   the admin views (:ticket:`27018`).
+
+* Fixed the ``migrate`` command to include already applied migrations state
```

Comment:
The idea is merely to try to ward off duplicate reports in case someone sees the behavior on 1.10 and doesn't realize it's been fixed. Not critical if it can't be done easily.